### PR TITLE
Add a warning if there is more than 1 result 

### LINKED ISSUE #26: Warn the user when we find multiple results (and select the first one automatically)
Temporary solution for #25 that we can add before implementing that ticket.

### PR DESCRIPTION
**Pull request by dhartwich1991**
Add a warning if there is more than 1 result found for a given version name + code.

Closes #26

### CODE REVIEW

File: lib/crep/model/crash_sources/hockeyapp_crash_source.rb [hunk 1]
```diff
@@ -53,6 +53,8 @@ def version(version:, build:)
 
       raise "No version was found for #{version})#{build})" unless filtered_versions.count > 0
 
+      puts "Multiple results for #{version}/#{build} were found, using the first one that was found" unless filtered_versions.count <= 1
```
Comment (BasThomas) [inline lib/crep/model/crash_sources/hockeyapp_crash_source.rb]: Can we explicitly mention what we mean by "the first"? We can simply print it there.

With that, we should also print which versions were found (and thus, which one we will use).
Also a good contender for #44, making the used results green and the unused results yellow :)

Comment (dhartwich1991) [reply lib/crep/model/crash_sources/hockeyapp_crash_source.rb]: I made the warning now print in yellow and look like the following: `Multiple results for 8.0.0j/1700800000 were found, using the first (with ID 179) one that was found
`

Comment (dhartwich1991) [reply lib/crep/model/crash_sources/hockeyapp_crash_source.rb]: In a future PR I will definitely  also print the other versions that were found and eventually a chooser where you can select the version you want to analyze.

**Comment by BasThomas** [comment]
Ah - we forgot to add this to the changelog. I'll draft a pull request 👍.

**Comment by q231950** [comment]
Craft it @BasThomas 👌

**Pull request merged**